transport.go

```diff
@@ -83,6 +83,10 @@ func rollbarError(logger ClientLogger, format string, args ...interface{}) {
 }
 
 func writePayloadToStderr(logger ClientLogger, payload map[string]interface{}) {
+	if _, ok := payload["access_token"]; ok {
```

Comment:
what sets "access_token" in the payload? why do we need to do it?

Reply:
yeah, that's why I suggested @rollbar-bborsits to get back to the customer and ask them to upgrade. They may be using a really old version.

Reply:
I asked the customer, but they have yet to reply. We had some internal miscommunication hence this PR.

Reply:
Any update from the customer about this stale PR @rollbar-bborsits ?

It seems not valid in general based on the previous comments

Reply:
@jmarronm Yes, you're right. This is not a valid use-case, so we can close this PR.